packages/utils/heading/page-heading.tsx

```diff
@@ -83,7 +83,10 @@ const PageHeading: React.FC<PageHeadingProps> = (props) => {
       style={style}
     >
       {showBreadcrumbs && (
-        <Split style={{ alignItems: 'baseline' }}>
+        <Split
+          style={{ alignItems: 'baseline' }}
+          className="mcgms-page-heading-breadcrumbs"
```

Comment:
no need here...
details page also looks okay to me... only it is needed for creation page and that we can do so by passing the class as a prop (which we did above in this PR).

Reply:
I feel with padding it looks better.

![image](https://user-images.githubusercontent.com/22158580/175281750-b625aa1f-d8e8-4ad8-a3ac-7370a81847ca.png)

also, it maintains uniformity, we can keep the padding. I have kept the exact amount of padding we have in various openshift repo - 

![image](https://user-images.githubusercontent.com/22158580/175282474-1adf4617-1b72-4654-b04a-2d94cbdc9d72.png)
![image](https://user-images.githubusercontent.com/22158580/175282759-a462e2a7-eebc-49d0-acd7-7cc18e69698c.png)

without padding - 

![image](https://user-images.githubusercontent.com/22158580/175281672-a5e86d64-df69-4612-85f9-0d3de2e37ea9.png)